# Description:
#   name: diag name
#   is_active: Force update or not
#   status: diag status set by dummy diag publisher "OK, Warn, Error, Stale"
#
# Note:
#
# default values are:
#   is_active: "true"
#   status: "OK"
---
/**:
  ros__parameters:
    required_diags:
      dummy_diag_empty: default
